configure-load-base-split.md

```diff
@@ -68,6 +74,18 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
     curl "http://ip:status_port/config"
     ```
 
+## 监控信息
```

Comment:
I prefer a method for how to get proper value for load-base-split region by metrics instead of meaning of metrics itself.

Reply:
Do you have a good idea?